src/audio.rs

```diff
@@ -34,31 +20,3 @@ pub struct SoundEffect;
 pub fn sound_effect(handle: Handle<AudioSource>) -> impl Bundle {
     (AudioPlayer(handle), PlaybackSettings::DESPAWN, SoundEffect)
 }
-
-#[derive(Resource, Debug, Reflect, Deref, DerefMut, Default)]
-#[reflect(Resource)]
-pub struct GlobalMusicVolumeScale(pub Volume);
-#[derive(Component, Debug, Reflect, Deref, DerefMut, Default)]
-#[reflect(Component)]
-pub struct UnscaledVolume(pub Volume);
-
-fn scale_mutated_music(
-    mut sinks: Query<(Mut<AudioSink>, &mut UnscaledVolume), (Changed<AudioSink>, With<Music>)>,
-    music_volume_scale: Res<GlobalMusicVolumeScale>,
-) {
-    for (mut sink, mut unscaled_volume) in &mut sinks {
-        let sink = sink.bypass_change_detection();
-        unscaled_volume.0 = sink.volume();
-        sink.set_volume(music_volume_scale.0 * unscaled_volume.0);
-    }
-}
-
-fn update_music_for_new_global_scale(
-    music_volume_factor: Res<GlobalMusicVolumeScale>,
-    mut sinks: Query<(&mut AudioSink, &UnscaledVolume), With<Music>>,
-) {
-    for (mut sink, unscaled_volume) in &mut sinks {
-        let sink = sink.bypass_change_detection();
-        sink.set_volume(music_volume_factor.0 * unscaled_volume.0);
-    }
-}
```

Comment:
Smt. like this might still be needed based on the note from [GlobalVolume](https://docs.rs/bevy/latest/bevy/audio/struct.GlobalVolume.html).

> Note: Changing [GlobalVolume](https://docs.rs/bevy/latest/bevy/prelude/struct.GlobalVolume.html) does not affect already playing audio.

Reply:
heck

Reply:
Opened an issue for this: https://github.com/bevyengine/bevy/issues/18952